### PR TITLE
Network switch tests

### PR DESCRIPTION
**Pull request by sadov**
Added tests for switching of app's networks and ports redirection:
```
eden test tests/eclient/ -v debug
```

### CODE REVIEW

**Review by rvs — APPROVED** [review]
LGTM

**Review by giggsoff — COMMENTED** [review]
Please see my comments

File: tests/eclient/testdata/nw_switch.txt [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+eden network create 10.1.0.0/24 -n n1
+#exec sleep 5
+eden network create 10.2.0.0/24 -n n2
+
+exec bash wait_net.sh n1 n2
+stdout '^n1\s'
+stdout '^n2\s'
+
+message 'Starting applications'
+eden pod deploy -v debug -n ping1 docker://itmoeve/eclient:latest -p 2223:22 --networks=n1
```
[the remark this answers is not in the document]
Comment (sadov) [reply tests/eclient/testdata/nw_switch.txt]: Done

File: tests/eclient/testdata/port_switch.txt [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+eden pod deploy -p 8027:80 docker://nginx -v debug -n nginx
+
+test eden.app.test -test.v -timewait 20m RUNNING nginx
+
+eden pod ps
+cp stdout pod_ps
+exec bash server_ip.sh nginx
+
+exec sleep 10
+exec -t 1m bash run_clent.sh
```
[the remark this answers is not in the document]
Comment (sadov) [reply tests/eclient/testdata/port_switch.txt]: OK -- added output of the script check.

**Pull request merged**